src/js/requestHandlers.js

```diff
@@ -85,12 +87,12 @@ sjrk.storyTelling.server.handleGetStory = function (request, dataSource, uploade
         fluid.transform(response.content, function (block) {
             if (block.blockType === "image") {
                 if (block.imageUrl) {
-                    block.imageUrl = uploadedFilesHandlerPath + "/" + block.imageUrl;
+                    block.imageUrl = uploadedFilesHandlerPath + path.sep + block.imageUrl;
```

Comment:
I believe that `path.sep` is incorrect in this function -- we're building URLs in this function, right? And the separator is always '/'.

Reply:
Thanks for catching this, I think this one tripped me up in 2017 as well. I got a bit overzealous with my replacement.